src/cdm/open_cdm_platform_common.h

```diff
@@ -18,7 +18,15 @@
 #define MEDIA_CDM_PPAPI_EXTERNAL_OPEN_CDM_CDM_OPEN_CDM_PLATFORM_COMMON_H_
 
 #include <string>
+#ifdef WPE
+#include <stddef.h>
+#include <limits.h>
+#include <stdint.h>
+typedef unsigned char      uint8;
+typedef unsigned int       uint32;
+#else	// Chrome
 #include "base/basictypes.h"
```

Comment:
I think this #ifdef can be avoided. A quick google returns this https://groups.google.com/a/chromium.org/forum/#!msg/chromium-dev/zFzvBFptJj4/9DbUeduq_iYJ

So it looked like Chrome were trying to move to stdint.h and away from base/basictypes.h.